Examples/javascript/array/runme.js

```diff
@@ -0,0 +1,7 @@
+var example = require("example");
+
+var arr = [1, 2, 3, 4, 5];
+
+console.log(example.sum1(arr, arr.length));
+console.log(example.sum2(arr));
+console.log(example.data3)
```

Comment:
Non-automated tests are much less useful, as we won't catch regressions right away (and probably not until somebody finds their SWIG-generated wrappers broke).

If you don't want to make code under `Examples/javascript` self-check its results, then please put this in the main test-suite under `Examples/test-suite` instead.